mkosi.build

```diff
@@ -20,11 +20,15 @@
 # This is a build script for OS image generation using mkosi (https://github.com/systemd/mkosi).
 # Simply invoke "mkosi" in the project directory to build an OS image.
 
+# If mkosi.builddir/ exists mkosi will set $BUILDDIR to it, let's then use it
+# as out-of-tree build dir. Otherwise, let's make up our own builddir.
+[ x"$BUILDDIR" = x ] && BUILDDIR=build
```

Comment:
This will terminate the script (because of `-e`). You need to use `||`. And "x" is so ugly — it was used because of very old incompatible shells...
```bash
  [ -n "$BUILDDIR" ] || BUILDDIR=build
```